tools/isolationchamber.c

```diff
@@ -19,10 +22,7 @@ void extra_create()
     set( "day_light" , PART_LIT );
     set( "night_light" , PART_LIT );
     set( InsideP, 1 );
-    set( "exits" , ([
-         "lab" : "/usr/gon/workroom.c",
-         ])
-        );
+    set( "exits" , ( [ "lab" : "/usr/gon/workroom.c", ] ) );
```

Comment:
I'm not super tied to either way. As you've said before, there's a LOT of variation on EotL and so I'm kind of picking up certain whitespace and padding examples depending on the code I'm reviewing.